### ISSUE TITLE
 ERR:GLOBAL:rtpp_module_if_ctor: can't dlopen(../modules/acct_csv/.libs/rtpp_acct_csv.so): ../modules/acct_csv/.libs/rtpp_acct_csv.so: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by africallshop** [sippy/rtpproxy#39]
Hello,

One error when starting rtpproxy 

 ERR:GLOBAL:rtpp_module_if_ctor: can't dlopen(../modules/acct_csv/.libs/rtpp_acct_csv.so): ../modules/acct_csv/.libs/rtpp_acct_csv.so: cannot open shared object file: No such file or directory

Regards

**Comment by shirsenduj** [comment]
We are also facing the same problem..can some one help on this?
Regards

**Comment by sobomax** [comment]
Yeah, sorry, it's some debug code. I'll tidy it up soon. Should be harmless. Thanks!
